src/action/ScaleByAreaAction.cpp

```diff
@@ -63,6 +66,7 @@ void ScaleByAreaAction::performAction(double time)
       }
     }
   }
+  _targetData->setSampleAtTime(time::Storage::WINDOW_END, _targetData->sample());
```

Comment:
Here, we can actually use the same strategy like for the mapping, because we only modify `_targetData` in-place. So there is no possibility for incompatible time meshes like in the summation action.

Reply:
> the same strategy like for the mapping

Which is?

Reply:
Took care of this. [73e0cea](https://github.com/precice/precice/pull/1612/commits/73e0ceada27ee10a3cd01857bba0057057bee2b7)